docs/classic-ui/images.md

```diff
@@ -26,15 +26,19 @@ We will use the image object as context in the following examples.
 
 In `/@@imaging-controlpanel` Plone allows you to configure which scales are available and what dimensions they should have. By default we have the following scales configured:
 
-* large 768:768
-* preview 400:400
-* mini 200:200
+* huge 1600:65536
+* great 1200:65536
+* larger 1000:65536
+* large 800:65536
+* teaser 600:65536
+* preview 400:65536
+* mini 200:65536
```

Comment:
65536 is very high. Are you sure about these?

Reply:
Yes, they came from plone.base.interfaces.controlpanel